packages/gatsby-plugin-sharp/src/gatsby-node.js

```diff
@@ -51,7 +52,18 @@ const { setPluginOptions } = require(`./plugin-options`)
 //       reportError(message || err.message, err, reporter)
 //     })
 
-exports.onPreBootstrap = ({ actions }, pluginOptions) => {
+// create the progressbar once and it will be killed in another lifecycle
+const finishProgressBar = () => {
+  const progressBar = getProgressBar()
+  if (progressBar) {
+    progressBar.done()
+  }
+}
+
+exports.onPostBuild = () => finishProgressBar()
+exports.onCreateDevServer = () => finishProgressBar()
+
+exports.onPreBootstrap = ({ actions, reporter }, pluginOptions) => {
```

Comment:
```suggestion
exports.onPreBootstrap = ({ actions }, pluginOptions) => {
```

we don't seem to use `reporter` in here